app/src/main/java/com/example/ana/exampleapp/Variables.java

```diff
@@ -35,6 +34,11 @@
      * Final class: to prevent someone from accidentally instantiating the class, we give it an
      * empty constructor.
      */
+    public void ipfromsystem(){
```

Comment:
Code Conventions should follow Oracle recommendations http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#367